packages/extension/src/manifest.json

```diff
@@ -10,30 +11,56 @@
     "48": "icons/48.png",
     "128": "icons/128.png"
   },
-  "__prod__permissions": [
+  "__prod__permissions": ["scripting"],
+  "__prod|chrome|opera|edge__host_permissions": [
     "https://daily.dev/",
     "https://*.daily.dev/",
     "https://dailynow.co/",
     "https://*.dailynow.co/"
   ],
-  "__dev__permissions": ["storage", "http://localhost/", "http://*.local.com/"],
-  "optional_permissions": ["topSites", "*://*/*"],
-  "content_security_policy": "script-src 'self' https://www.google-analytics.com; object-src 'self'",
-  "__firefox__content_security_policy": "object-src 'self'",
-  "browser_action": {
+  "__firefox|prod__permissions": [
+    "scripting",
```

Comment:
This is a additional request right?
Why do we need it for firefox (and chrome for that matter)

Reply:
This is needed so we can make the request to register the companion content script: https://github.com/dailydotdev/apps/pull/2332/files/d9b5d2c91912ca948c32ea17e0a4532d336e3fce#diff-9d3bc2384cfa13933a9b00959c4da1058f54cc6338b656556dab709bc868ba5eR42

this has changed from `browser.contentScripts.register` which was used before and is no longer supported in V3.

Without this, we get the following error:
```
TypeError: D().scripting is undefined
```

We could use different APIs, but if this works on V2 as well, not sure if it's worth it to have an `if/else` for this

Reply:
It does not trigger any permission popups?

Reply:
> It does not trigger any permission popups?

No new permission pop-ups are generated by this. We can validate this again through a test release of the firefox extension

Reply:
Fine by me.